Project/ProjectSettings/ProjectVersion.txt

```diff
@@ -1 +1 @@
-m_EditorVersion: 2018.4.24f1
```

Comment:
Is this necessary or is it just to make our lives easier?

Reply:
accidental commit, will undo